<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StegTypeSelectionWindow</class>
 <widget class="QWidget" name="StegTypeSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Steganography Type Selection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #282a36;
}

QLabel {
    color: #f8f8f2;
    font-size: 16px;
    font-weight: bold;
    margin-bottom: 10px;
}

QPushButton {
    background-color: #44475a;
    color: #f8f8f2;
    border: none;
    border-radius: 5px;
    padding: 10px;
    font-size: 14px;
    margin-bottom: 10px;
}

QPushButton:hover {
    background-color: #6272a4;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Select Steganography Type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
    <item>
    <widget class="QPushButton" name="btnBack">
    <property name="text">
    <string>Home</string>
    </property>
    </widget>
    </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnText">
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnImage">
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBinary">
     <property name="text">
      <string>Binary</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>